src/mapml-viewer.js

```diff
@@ -234,6 +235,37 @@ export class MapViewer extends HTMLElement {
         this.dispatchEvent(new CustomEvent('createmap'));
       }
     }
+
+    /*
+      1. only deletes aria-label when the last (only remaining) map caption is removed
+      2. only deletes aria-label if the aria-label was defined by the map caption element itself
```

Comment:
Will need to rethink how to handle multiple `map-captions` as per our discussion from this morning:
* Only use the first map-caption?
*  When multiple map-captions are added, append them to the aria-label - similar to what's done with fig-captions?